src/reports/components/report-sections/no-failed-instances-congrats.scss

```diff
@@ -2,35 +2,24 @@
 // Licensed under the MIT License.
 @import '../../../common/styles/colors.scss';
 
-.report-congrats {
-    position: relative;
-}
-
-.report-congrats-screen {
-    top: 181px;
-    position: absolute;
-    width: 488px;
-    left: 105px;
-    height: 295px;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-}
-
 .report-congrats-message {
-    width: 364px;
-    text-align: center;
+    word-break: break-all;
+    word-wrap: break-word;
```

Comment:
- Should `word-wrap` be `overflow-wrap` instead? `word-wrap` is an alias per [this MDN page](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap) and it was hard to look this up until I figured this out.
- In the screenshot, the text is wrapping in the middle of "word", which is not usual behavior. This seems to be due to the use of `break-all`. Removing the `word-break: break-all` seems to restore usual behavior. Is there a scenario where we need the `break-all`?

Reply:
- sure we can, as I know word-wrap and overflow-wrap are identical.
- Yeah that's what Fer wants, and that's what we do in the report.

Reply:
I'm also a bit concerned about breaking in the middle of a word, it would be nice if we can avoid doing that.

Reply:
As I mentioned before replying to the second point 
- Yeah that's what we want, I checked with Fer , and that's what we do in the report.